server/src/index.mjs

```diff
@@ -83,6 +84,8 @@ app.use(bodyParser.json({ limit: '1mb' })); // to support JSON-encoded bodies
 app.use(bodyParser.urlencoded({             // to support URL-encoded bodies
   extended: true
 }));
+app.use(cors());
+app.options('*',cors());
```

Comment:
So allowing all origins by default?

Reply:
Yes; seems fine given it's a local/development tool.